scene/gui/code_edit.cpp

```diff
@@ -443,8 +443,8 @@ void CodeEdit::gui_input(const Ref<InputEvent> &p_gui_input) {
 
 		if (symbol_lookup_on_click_enabled) {
 			if (mm->is_command_or_control_pressed() && mm->get_button_mask().is_empty()) {
-				symbol_lookup_pos = get_line_column_at_pos(mpos);
-				symbol_lookup_new_word = get_word_at_pos(mpos);
+				symbol_lookup_pos = get_line_column_at_pos(mpos, false);
+				symbol_lookup_new_word = get_word_at_pos(mpos, false);
```

Comment:
Please add `false` for `get_word_at_pos()` here too:

https://github.com/godotengine/godot/blob/367c157beac0b26f386a3d33f9b92f057adf2aeb/scene/gui/code_edit.cpp#L457-L458